### PR TITLE
[REG-1246] docs: return 'get test domain on polygonscan page'

### PR DESCRIPTION
**Pull request by roman-gnativ**
## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: getting-started/test-domains/polygonscan.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+0x4118ebbd893ecbb9f5d7a817c7d8039c1bd991b56ea243e2ae84d0a1b2c950a7 = 'blockchain';
+0x042fb01c1e43fb4a32f85b41c821e17d2faeac58cfc5fb23f80bc00c940f85e3 = 'bitcoin';
+0x5c828ec285c0bf152a30a325b3963661a80cb87641d60920344caf04d4a0f31e = '888';
+0xb5f2bbf81da581299d4ff7af60560c0ac854196f5227328d2d0c2bb0df33e553 = 'dao';
+0xa18784bb78ee0f577251fb21ad5cac7a140ab47e9414e3c7af5125e3e1d28923 = 'klever';
+0xfdb51f7f56d9b1149db5ce99afcf60dda4416fd6fb8dc0649fec13cd03e1803e = 'hi';
+0x2acf53593112265ba651274f0e33a6b3fe86f92bbee4d39211540592fde6b0f3 = 'kresus';
+0xed9ce6b49a0e2c56c57c86795b131bd6df792312183994c3cf3de1516cfe92d6 = 'polygon';
+0xaf4d61a6acdda7379d6f90207923aeb080994d0f1aaba4b8b24a14f965b2bb7c = 'anime';
+0x57ea207b3238a69d8a9a53252de172f2ceae1d856008b13208a203867a707c78 = 'manga';
```
Comment (nickshatilo) [inline getting-started/test-domains/polygonscan.md:40]: probably make sense to use the full list

File: getting-started/test-domains/polygonscan.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![Enter data for 'claim to records' for Polygonscan domain (Layer2)](/images/polygonscan-claim-to-records.png)
+
+<figcaption>Enter data for 'claim to records' for Polygonscan domain (Layer2)</figcaption>
+</figure>
+
+- Click the **Write** button to start the transaction.
+
+:::success Congratulations!
+You're the new proud owner of a test domain on Polygon Mumbai testnet. Happy hacking!
```
Comment (nickshatilo) [inline getting-started/test-domains/polygonscan.md:62]: `mumbai`

File: getting-started/test-domains/polygonscan.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+title: Get a Free Polygon Test Domain Guide | Unstoppable Domains Developer Portal
+description: This guide covers the process of obtaining a free test domain, through direct smart contract calling on Polygonscan.
+---
+
+# Get a Free Polygon Test Domain Guide
+
+<embed src="/snippets/_test-domain-explain.md" />
+
+:::info
+[Polygon faucet](https://faucet.polygon.technology) offers free money in the form of matic tokens for domains minted on Polygon Testnet.
```
Comment (tirish) [inline getting-started/test-domains/polygonscan.md:11]:
```suggestion
[Polygon faucet](https://faucet.polygon.technology) offers free MATIC tokens for Polygon Testnets. You will need some MATIC to interact with our smart contracts and claim a free test domain.
```

File: getting-started/test-domains/polygonscan.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+You can locate the domain ending [namehash](../domain-registry-essentials/namehashing.md) using the following list:
+
+```bash
+"0x58872a8ca9414ae0f3ab850f51efaf4c77b647d903f9ec89a03d3018bb42e4fa = 'clay';
+"0x0f4a10a4f46c288cea365fcf45cccf0e9d901b945b9829ccdb54c10dc3cb7a6f = 'crypto';
+"0x89e68000b532da79f9a0d4b8416eaa4f64294236ab233205343453d5bec22933 = 'austin';
+"0x17936dd2b9c99a1e1bfa039dd7d40118f3ed535fcd9c8cda23c8070ffb4c2264 = 'pudgy';
+"0x9bf15275eb85f2e69308f3dbfbe0f94cade4f05c2aa24ba7b9eba5ad2fb1a2cb = 'altimist';
+"0xa264639c3dbff58a1ded5efcf724e50a9c7847ff8197c87c00d009fbba2a3720 = 'go';
+"0x2ac5a394f941733376764fbf4c4b2933098b23a405bd8c55c9a8d32e2c8fb696 = 'realm';
```
Comment (tirish) [inline getting-started/test-domains/polygonscan.md:33]: We are not officially live with `realm`, so we should exclude it.

Are these anywhere in the UNS repo we could just link to instead so we don't have to maintain this list?

**Review by tirish — APPROVED** [review]

**Pull request merged**